projects/plugins/jetpack/modules/wordads/class-wordads.php

```diff
@@ -956,7 +971,7 @@ public function get_house_ad( $unit = 'mrec' ) {
 	 * @since 8.7
 	 */
 	public static function get_watl_ad_html_tag( string $slot_type ): string {
-		return "<div class=\"wordads-tag\" data-slot-type=\"$slot_type\" style=\"display: none;\"></div>";
+		return "<div class=\"wordads-tag\" data-slot-type=\"$slot_type\" data-tag-origin=\"jetpack\" style=\"display: none;\"></div>";
```

Comment:
I added this as metadata to show where the tag was injected from. It's open to suggestions e.g name change